docs/pages/eas-build-custom-builds/custom-build-functions.mdx

```diff
@@ -0,0 +1,288 @@
+---
+title: Custom JavaScript/TypeScript functions
+description: Learn how to create and use custom JavaScript/TypeScript functions in your EAS Build custom builds configuration.
+---
+
+import { Terminal } from '~/ui/components/Snippet';
+import { Step } from '~/ui/components/Step';
+import ImageSpotlight from '~/components/plugins/ImageSpotlight';
+import { BoxLink } from '~/ui/components/BoxLink';
+import { GithubIcon } from '@expo/styleguide-icons';
+
+Custom functions are a great way to extend the functionality of EAS Build custom builds. You can use them to write reusable pieces of custom build config in a very powerful and convenient way.
+
+<Step label="1">
+
+## Creating custom JavaScript/TypeScript functions
+
+The easiest way to create a custom function is to use the `create-eas-build-function` CLI tool.
+
+By running the following command, from the same as your `eas.json` file, you will be able to create a new custom TypeScript function:
```

Comment:
```suggestion
The easiest way to create a custom function is to use the `create-eas-build-function` CLI tool. By running the following command from the same directory as your **eas.json** file, you can create a new custom TypeScript function:
```

- [File name bold](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md#filenames-as-bold-text)
- Felt like there was some context missing and confusing when: "...By running the following command, from the same as your `eas.json` file...". Did not understand what the "same" is referencing too.